packages/kbn-eslint-plugin-imports/src/helpers/running_in_editor.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const RUNNING_IN_EDITOR =
+  // vscode sets this in the env for all workers
+  !!process.env.VSCODE_CWD ||
+  // MacOS sets this for intellij processes, not sure if it works in webstorm but we could expand this check later
+  process.env.__CFBundleIdentifier === 'com.jetbrains.intellij';
```

Comment:
Maybe we can use here `!!process.env.__CFBundleIdentifier?.includes('com.jetbrains.')` as I'm 99% sure this on webstorm will be com.jetbrains.webstorm